# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/inkscape/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- inkscape-1.4_2024-10-09_e7c3feb100/src/util/gobjectptr.h.vanilla	2025-04-29 01:04:46.220691361 +0300
+++ inkscape-1.4_2024-10-09_e7c3feb100/src/util/gobjectptr.h	2025-04-29 01:05:11.656440904 +0300
@@ -20,7 +20,7 @@
     GObjectPtr() = default;
     explicit GObjectPtr(T *p, bool add_ref = false) : _p(p) { if (add_ref) _ref(); }
     GObjectPtr(GObjectPtr const &other) : _p(other._p) { _ref(); }
-    GObjectPtr &operator=(GObjectPtr const &other) { if (&other != this) { _unref(); _p = other.p; _ref(); } return *this; }
+    GObjectPtr &operator=(GObjectPtr const &other) { if (&other != this) { _unref(); _p = other._p; _ref(); } return *this; }
     GObjectPtr(GObjectPtr &&other) noexcept : _p(other._p) { other._p = nullptr; }
     GObjectPtr &operator=(GObjectPtr &&other) { if (&other != this) { _unref(); _p = other._p; other._p = nullptr; } return *this; }
     ~GObjectPtr() { _unref(); }
